docs/reference/commandline/swarm_init.md

```diff
@@ -118,6 +119,12 @@ for example `--advertise-addr eth0:2377`.
 Specifying a port is optional. If the value is a bare IP address or interface
 name, the default port 2377 will be used.
 
+### `--datapath-addr`
+
+This flag specifies the address or the interface that is going to be used for
+data path traffic in global scope drivers. If unspecified, Docker will use the
+same IP address and interface that is used for the advertise address.
```

Comment:
Please clarify in the documentation that this is an address that's used by other nodes, rather than one used for local binding.